horace_core/horace_off.m

```diff
@@ -7,22 +7,26 @@
 % T.G.Perring
 
 % root directory is assumed to be that in which this function resides
-rootpath = fileparts(fileparts(which('horace_init')));
+global horace_path
```

Comment:
Oh actually... should this not use the `paths` class?

Reply:
Done.